packages/tests/src/utils/nodes.ts

```diff
@@ -0,0 +1,114 @@
+import { waitForRemotePeer } from "@waku/core";
```

Comment:
why do we need to create a new file that seems like a duplication of https://github.com/waku-org/js-waku/blob/04bd518210a709b3b01ada5be947e653912201ba/packages/tests/tests/filter/utils.ts#L70

if we need that - let's move